Sources/Kumo/Kumo.docc/Kumo.md

```diff
@@ -0,0 +1,13 @@
+# ``Kumo``
+
+<!--@START_MENU_TOKEN@-->Summary<!--@END_MENU_TOKEN@-->
+
+## Overview
+
+<!--@START_MENU_TOKEN@-->Text<!--@END_MENU_TOKEN@-->
+
+## Topics
+
+### <!--@START_MENU_TOKEN@-->Group<!--@END_MENU_TOKEN@-->
+
+- <!--@START_MENU_TOKEN@-->``Symbol``<!--@END_MENU_TOKEN@-->
```

Comment:
got some placeholders here—not sure if we should just leave them

Reply:
They don't technically appear in the Developer Documentation view but it might get annoying for people looking at the markdown directly.

Reply:
That's ok then, I wouldn't worry about dealing with this